docs/nightly/en/user-guide/query-data/view.md

```diff
@@ -0,0 +1,122 @@
+# View
+
+In SQL, a view is a virtual table based on the result set of an SQL statement. It contains rows and columns just like a real table. In the following situations, we can use views:
+
+* Simplifying complex queries, avoiding the need to repeatedly write and send complex statements for every query.
+* Granting read permissions to specific users while restricting access to certain columns and rows to ensure data security and isolation.
+
+A view is created with the `CREATE VIEW` statement.
+
+## View examples
+
+```sql
+CREATE VIEW cpu_monitor AS
+    SELECT cpu, host, ts FROM monitor;
+```
```

Comment:
when new data is inserted into the `monitor`, will the data in the `cpu_monitor` view be updated synchronously?

Reply:
There is no update happening, view is just a represent of a query.